seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/catalog/oceanbase/OceanBaseMySqlCatalog.java

```diff
@@ -201,8 +209,7 @@ public CatalogTable getTable(String sqlQuery) throws SQLException {
         try (Statement statement = defaultConnection.createStatement();
                 ResultSet resultSet = statement.executeQuery(sqlQuery)) {
             ResultSetMetaData metaData = resultSet.getMetaData();
-            return CatalogUtils.getCatalogTable(
-                    metaData, new OceanBaseMySqlTypeMapper(typeConverter), sqlQuery);
+            return getCatalogTable(metaData, defaultConnection.getMetaData(), sqlQuery);
```

Comment:
shall we update https://github.com/apache/seatunnel/blob/e4e12245b90e994bce27ea90ac044e722888a8ae/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/catalog/oceanbase/OceanBaseMySqlCatalog.java#L130 too?

Reply:
No, I reused it